### ISSUE TITLE
Support for Android app bundles

### ISSUE THREAD

**Issue by GForceWeb** [bitrise-steplib/steps-ionic-archive#70]
### Issue description

Google will require Android App Bundles for new apps starting August 2021. With that deadline approaching, I'm trying to adjust my workflows to use AAB files instead of APKs.

I've added --packageType=bundle parameter to my Ionic CLI build command which allows an aab file to be compiled successfully but the step doesn't recognise it and returns an error because there's no APK file.

### Bitrise info

- Build URL: https://app.bitrise.io/build/8f17b587-2c4f-4f75-bd8e-7c70ba003983#?tab=log
- [Bitrise Support](https://devcenter.bitrise.io/troubleshooting/enabling-bitrise-support-user/) enabled: YES
- Log:

```BUILD SUCCESSFUL in 1m 43s
43 actionable tasks: 43 executed
Built the following bundle(s): 
	/Users/vagrant/git/platforms/android/app/build/outputs/bundle/debug/app-debug.aab
Collecting android outputs
No apk generated```

**Comment by mateherber** [comment]
Hi @GForceWeb ,

We released the `2.3.0` version of the step that should add proper `aab` support to the step.

**Comment by GForceWeb** [comment]
Great, thanks for that @mateherber. Will Check that out
